units/src/amount/signed.rs

```diff
@@ -123,14 +156,35 @@ impl SignedAmount {
     /// or with [`fmt::Display`].
     ///
     /// If you want to parse only the amount without the denomination, use [`Self::from_str_in`].
+    ///
+    /// # Errors
+    ///
+    /// If the amount is too big (positive or negative) or too precise.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bitcoin_units::{amount, SignedAmount};
+    /// let amount = SignedAmount::from_str_with_denomination("0.1 BTC")?;
+    /// assert_eq!(amount, SignedAmount::from_sat(10_000_000));
+    /// # Ok::<_, amount::ParseError>(())
+    /// ```
     pub fn from_str_with_denomination(s: &str) -> Result<SignedAmount, ParseError> {
         let (amt, denom) = split_amount_and_denomination(s)?;
         SignedAmount::from_str_in(amt, denom).map_err(Into::into)
     }
 
-    /// Express this [`SignedAmount`] as a floating-point value in the given denomination.
+    /// Express this [`SignedAmount`] as a floating-point value in the given [`Denomination`].
```

Comment:
```suggestion
    /// Expresses this [`SignedAmount`] as a floating-point value in the given [`Denomination`].
```

Reply:
There are a few more instances of missing 'es' - I don't know the grammar term for that.

Reply:
Done